src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -476,28 +505,71 @@ class SignUpModalController {
   }
 
   injectErrorMessages (errors = this.signUpErrors) {
+    this.clearInjectedErrorMessages()
+
+    if (!errors) {
```

Comment:
```suggestion
    if (!errors.length) {
```
We should check if errors length is falsey.

When testing `!['a', 'b']` it returns `false`.

Reply:
The goal here was to handle the case where errors was `null` or `undefined`. It was an alternative to `errors?.forEach`. But since I don't see a way that `errors` could not be an array, I'll just remove this check.

Reply:
I like the check, as it means we don't run the entire function on each step if there are no errors.